lib/src/model/documentation_comment.dart

```diff
@@ -675,8 +672,8 @@ mixin DocumentationComment
   /// Match group 4 is the unquoted arg, if any.
   static final RegExp _argPattern = RegExp(r'([a-zA-Z\-_0-9]+=)?' // option name
       r'(?:' // Start a new non-capture group for the two possibilities.
-      r'''(["'])((?:\\{2})*|(?:.*?[^\\](?:\\{2})*))\2|''' // with quotes.
-      r'([^ ]+))'); // without quotes.
+      r'''(["'])((?:[^\\\r\n]|\\.)*?)\2|''' // with quotes.
+      r'(\S+))'); // without quotes.
```

Comment:
This is a change. It didn't seem reasonable to accept newlines or tabs as part of an unquoted argument. Matching non-whitespace seemed better than just non-space-char.